docs/upgrade_to_2.0.rst

```diff
@@ -16,7 +16,7 @@ C++11 no longer supported
 =========================
 
 libtorrent 2.0 requires at least C++-14. To build with boost build, specify the
-C++ version using the ``cxxstd=14`` build feature (14 is the default).
+C++ version using the ``cxxstd=17`` build feature (14 is the default).
```

Comment:
I don't think this patch will land ahead of the 2.0 cut, so this change should not bed made